tests/test_workflow.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+import warnings
+
+from y0.dsl import Variable
+from y0.examples import frontdoor_backdoor
+from y0.examples.frontdoor import generate_data_for_frontdoor
+from y0.examples.frontdoor_backdoor import generate_data_for_frontdoor_backdoor
+from y0.graph import NxMixedGraph
+
+from src.eliater.examples import continuous, multi_med, multi_med_confounder
```

Comment:
what's up with all of these `src.` prefixes? Remove all of these from each file - the point of importing things is so we know that it always works uniformly, no matter how it's installed

Reply:
Done